mix.exs

```diff
@@ -22,4 +23,19 @@ defmodule MixUnusedDiscoveries.MixProject do
       {:mock, "~> 0.3.7", only: :test}
     ]
   end
+
+  defp aliases do
+    [
+      check: [
+        "compile --all-warnings --ignore-module-conflict --warnings-as-errors --debug-info",
+        "format --check-formatted mix.exs \"lib/**/*.{ex,exs}\" \"test/**/*.{ex,exs}\" \"priv/**/*.{ex,exs}\" \"config/**/*.{ex,exs}\"",
+        "deps.unlock --check-unused",
+        "credo -a --strict",
+        "dialyzer"
+      ],
+      "format.all": [
+        "format mix.exs \"lib/**/*.{ex,exs}\" \"test/**/*.{ex,exs}\" \"priv/**/*.{ex,exs}\" \"config/**/*.{ex,exs}\""
+      ]
```

Comment:
Questo puoi rimuoverlo. `mix format` formatta tutto quello che è definito in `inputs` nel file `.formatter.exs`.
Quindi dovrebbe già formattare tutto.